### PR TITLE
Traffic light launching procedure updated

### PR DESCRIPTION
**Pull request by rafaelfroehlich**
Because of the wiring, I had to change a bit the code in Software, but for now it should work
    I will remove the traffic light part from the robotarium book
@tanij

### CODE REVIEW

**Comment by AmaurX** [comment]
Pull request opened by me, not rafael...

File: book/opmanual_duckietown/13_traffic_lights_assembly.md [hunk 1]
```diff
@@ -480,18 +490,11 @@ See rc-control
...
-
-    duckiebot $ cd ~/duckietown
-    duckiebot $ git checkout TL-megacity
-
-After this step, launch traffic light node.
+### Launch Traffic Lights {#dt-ops-tl-launch status=ready}
+To download and run the trafficlight docker image, simply run:
 
-    duckiebot $ make traffic-light
+    docker -H ![hostname].local run -dit --privileged --name trafficlight --network=host duckietown/traffic-lights
```
Comment (liampaull) [inline book/opmanual_duckietown/13_traffic_lights_assembly.md]: there should be a master18 or master19 tag

Comment (AmaurX) [reply book/opmanual_duckietown/13_traffic_lights_assembly.md]: will add that

Comment (AmaurX) [reply book/opmanual_duckietown/13_traffic_lights_assembly.md]: done

**Pull request merged**